client/webserver/webserver.go

```diff
@@ -121,16 +127,16 @@ out:
 // WebServer is a single-client http and websocket server enabling a browser
 // interface to the DEX client.
 type WebServer struct {
-	ctx       context.Context
-	core      clientCore
-	addr      string
-	srv       *http.Server
-	html      *templates
-	indent    bool
-	mtx       sync.RWMutex
-	authToken string
-	syncers   map[string]*marketSyncer
-	clients   map[int32]*wsClient
+	ctx            context.Context
+	core           clientCore
+	addr           string
+	srv            *http.Server
+	html           *templates
+	indent         bool
+	mtx            sync.RWMutex
+	validAuthToken string
```

Comment:
Does the 'valid' prefix really add any clarity? Are we also tracking invalid auth tokens?

Reply:
I think it does, because even though we don't track invalid auth tokens, we do not invalidate them either. Might leave one wondering, why is "this" the `authToken` when it is different from the previously issued `authToken`?

More precisely, `WebServer.authorize()` ignores any value currently set at `WebServer.authToken`, generates a new token and overrides the previously set token. Begs the question, why generate and set a new auth token when one is already set? Naming it `validAuthToken` explains (to me at least) that the previous token is no longer valid and the newly generated token is the valid one.